awx/ui_next/src/components/Lookup/InventoryLookup.jsx

```diff
@@ -29,6 +29,7 @@ function InventoryLookup({
   fieldId,
   promptId,
   promptName,
+  overrideIsDisabled,
```

Comment:
nitpick: The convention we usually follow would make this `isOverrideDisabled`

Reply:
@jakemcdermott, updated per comment.